kep/61-real-load-aware-scheduling/61-real-load-aware-scheduling.md

```diff
@@ -248,7 +252,14 @@ According to the scores we have, node `N3` will be selected. The utilization fra
 <img src="images/image7.png" alt="after-placement" width="600" height="150"/>
 
 
-If we plot these in mu-sigma plots, we can see the placement automatically pushes the utilization of nodes toward the diagonal line sigma = 1 - mu.
+If we plot these in mu-sigma plots, we can see the placement automatically pushes the utilization of nodes toward the diagonal line sigma = 1 - mu. The 1 here indicates 100% of utilization. What can be configured is a coefficient `ita`, which indicates `mu + ita x sigma <= 100 %` and we choose `ita = 1` here. `ita` here models the confidence of usage not exceeding the node capacity under the assumption that the actual usage follows Gaussian distribution and follows the [68-96-99.5 rule](https://en.wikipedia.org/wiki/68%E2%80%9395%E2%80%9399.7_rule).
+
+So when `ita` gets different values, we get different confidence of not exceeding the capacity.
+
+- `ita = 1`, we have a 68% chance that the actual usage is within the node capacity.
+- `ita = 2`, we have a 95% chance that the actual usage is within the node capacity.
+- `ita = 3`, we have a 99.7% chance that the actual usage is within the node capacity.
+By default, we choose `ita = 1` as we would like to improve the overall utilization.
```

Comment:
This is for discussion: https://github.com/kubernetes-sigs/scheduler-plugins/pull/61#discussion_r502649997